atuin-client/src/database.rs

```diff
@@ -138,10 +156,35 @@ impl Sqlite {
     }
 
     async fn save_raw(tx: &mut sqlx::Transaction<'_, sqlx::Sqlite>, h: &History) -> Result<()> {
-        sqlx::query(
-            "insert or ignore into history(id, timestamp, duration, exit, command, cwd, session, hostname, deleted_at)
-                values(?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9)",
-        )
+        sqlx::query(indoc::indoc! {
```

Comment:
added [`indoc`](https://crates.io/crates/indoc) to format query to more readable way, while removing unnecessary spaces from it.